tests/boost/test_DataType.cpp

```diff
@@ -16,7 +16,6 @@ BOOST_AUTO_TEST_CASE(test1) {
     std::cout << "    uint32:      " << uint32 << "\n";
     std::cout << "    uint64:      " << uint64 << "\n";
     std::cout << "    int4x2:      " << int4x2 << "\n";
-    std::cout << "    int4x2chime: " << int4x2chime << "\n";
```

Comment:
I do not understand this line? It seems unrelated and probably unintended.

Reply:
This was causing the test to fail (unable to compile with the line). I think there might be some uncommitted code that made it work?

Reply:
This enum constant is now called `int4x2_swapped_withoffset` instead. Can you use this name instead of removing the line?